securedrop_client/gui/widgets.py

```diff
@@ -215,16 +588,12 @@ def __init__(self, parent):
         self.source_list = SourceList(left_column)
         left_layout.addWidget(self.source_list)
 
-        self.error_status = QLabel('')
-        self.error_status.setObjectName('error_label')
-        left_layout.addWidget(self.error_status)
-        self.error_status.hide()
-
         self.layout.addWidget(left_column, 4)
 
         self.view_layout = QVBoxLayout()
         self.view_layout.setContentsMargins(0, 0, 0, 0)
         self.view_holder = QWidget()
+        self.view_holder.setStyleSheet('background: #fff;')
```

Comment:
We may want to come up with a standard practice of where we place CSS. Right now it seems some is in class constants and some is inline as strings. To help us find CSS later, we might want to make all of them class constants so we have a better idea where to look. This doesn't need to be fixed here, just thinking out loud.

Reply:
This is a good point, I think it will also help with maintainability and decoupling of style/functionality.

Reply:
So I decide to move all of our CSS to a class string var for each widget that we want to style. I also started labeling our widgets with object names to be used in CSS. This is a first step in moving out our CSS into CSS files.

Reply:
I might have missed some areas in the code, but will slowly be refactoring each widget in this way. Take a look at all of the new widget classes that I created for example.